frontend/packages/knative-plugin/src/components/functions/FunctionsList.tsx

```diff
@@ -1,15 +1,41 @@
 import * as React from 'react';
-import { EmptyState, EmptyStateVariant } from '@patternfly/react-core';
+import { EmptyState, EmptyStateIcon, EmptyStateVariant, Title } from '@patternfly/react-core';
 import { useTranslation } from 'react-i18next';
+import { getImageForIconClass } from '@console/internal/components/catalog/catalog-item-icon';
 import { Table, TableProps } from '@console/internal/components/factory';
 import ServiceHeader from '../services/ServiceHeader';
+import { CreateActionDropdown } from './CreateActionDropdown';
 import FunctionRow from './FunctionRow';
 
+import './FunctionsPage.scss';
+
 const FunctionsList: React.FC<TableProps> = (props) => {
   const { t } = useTranslation();
+  const functionImage = () => (
+    <img
+      className="odc-functions__empty-list__image"
+      src={getImageForIconClass('icon-serverless-function')}
+      alt=""
+    />
+  );
```

Comment:
Nit: Can you move this above FunctionsList and call the component `FunctionIcon`, with an upper case F?